docs/layers/trips-layer.md

```diff
@@ -40,8 +40,9 @@ const App = ({data, viewport}) => {
   const layer = new TripsLayer({
     id: 'trips-layer',
     data,
+    getPath: d => d.waypoints.map(p => p.coordinates),
```

Comment:
maybe update the comment describing the data format to 3D instead of 2D?

Reply:
Where do we describe the data format as 2d?

Reply:
before layer construction, line 27 - line 39

Reply:
This format is allowed. We also point users to the PathLayer doc which describes all the path formats that are supported.